frontend/src/components/BaseDashboard/components/FilterColumn/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useRef, useState } from "react";
+import PropTypes from "prop-types";
+import "./FilterColumn.scss";
+import useOutsideClick from "../../../useOutsideClick";
+
+const FilterColumn = ({ name, values, handleSelect, selectedValues }) => {
+  const [isVisible, setIsVisible] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+  const [searchResults, setSearchResults] = useState([]);
+  const ref = useRef();
+  values = values = values.filter((x, i, a) => a.indexOf(x) === i);
+
+  useOutsideClick(ref, () => setIsVisible(false));
+
+  useEffect(() => {
+    setSearchResults((values = values.filter((x, i, a) => a.indexOf(x) === i)));
```

Comment:
don't assign to props

Reply:
this also basically sets searchResults to undefined (you are assigning an assignment )